### PR TITLE
Colors in Classifications 

### PR DESCRIPTION
**Pull request by KodeurFou**
hi @mcoughlin with @celiune and @Adiman007 we've worked on this long over due task :point_right: :point_left: . 
It's the PR for the colors in classifications tasks, we added only few classifications in our rejected / comfirmed dictionnaries but we can make another update to the PR that modifies the dictionnaries if you want to add more classifications NOR colors.

### CODE REVIEW

**Comment by mcoughlin** [comment]
@KodeurFou I think these colors / classes need to go in the config. They are very particular to the instance of SkyPortal you are deploying.

**Review by Theodlz — CHANGES_REQUESTED** [review]
Hi @KodeurFou, I think your last commit belongs to another PR as it differs greatly from the purpose of this one. Please reverse it or simply delete the changes and recommit.

I agree with @mcoughlin, these colors should be configurable somehow. In the config makes sense for a first PR.

File: static/js/components/ShowClassification.jsx [hunk 1]
```diff
@@ -99,6 +112,23 @@ const ClassificationRow = ({ classifications }) => {
 
   const classification = classifications[0];
 
+  const confirmed_classes = ["GO GRANDMA", "GO GRANDMA (HIGH PRIORITY)"];
+  const rejected_classes = ["STOP GRANDMA"];
```
Comment (profjsb) [inline static/js/components/ShowClassification.jsx:116]: We should not be hard coding anything with GRANDMA here. You can add these via an environment variable in the config OR have these defined in a table.

**Review by profjsb — CHANGES_REQUESTED** [review]

**Comment by Adiman007** [comment]
@KodeurFou you pushed your code on the wrong branch, I think

**Pull request closed without merging**